samples/http/README.md

```diff
@@ -10,74 +10,53 @@ categories: ["Samples"]
 This AutoKitteh project demonstrates 2-way usage of HTTP, with AutoKitteh
 webhooks and the Python [requests](https://requests.readthedocs.io/) library.
 
-## API Documentation
+API Documentation:
 
 - https://docs.autokitteh.com/integrations/http/python
 - https://docs.autokitteh.com/integrations/http/events
 
-## Setup Instructions
+## How It Works
 
-1. Install and start a
-   [self-hosted AutoKitteh server](https://docs.autokitteh.com/get_started/quickstart),
-   or use AutoKitteh Cloud
+1. Send Authenticated Requests - Makes HTTP requests with credentials using Basic Auth and Bearer Token
+2. Test Authentication - Tests successful and failed authentication attempts
+3. Display Results - Shows response codes, headers, and messages from the server
 
-2. Run these commands to deploy this project's manifest file:
+## Cloud Usage
 
-   ```shell
-   git clone https://github.com/autokitteh/kittehub.git
-   ak deploy --manifest kittehub/samples/http/autokitteh.yaml
-   ```
-
-3. Look for the following lines in the output of the `ak deploy` command, and
-   copy the URL paths for later:
+1. Copy the webhook URL from the "Triggers" tab (see the [instructions here](https://docs.autokitteh.com/get_started/deployment#webhook-urls))
+2. Deploy the project
 
-   ```
-   [!!!!] trigger "..." created, webhook path is "/webhooks/..."
-   ```
-
-> [!TIP]
-> If you don't see the output of `ak deploy` anymore, you can run these
-> commands instead, and use the webhook slugs from their outputs:
->
-> ```shell
-> ak trigger get receive_http_get_or_head --project http_sample -J
-> ak trigger get receive_http_post_form --project http_sample -J
-> ak trigger get receive_http_post_json --project http_sample -J
-> ak trigger get send_requests --project http_sample -J
-> ```
-
-## Usage Instructions
+## Trigger Workflow
```

Comment:
This project has multiple webhooks, and the "Trigger Workflow" section needs to distinguish between them. See `samples/hubspot/README.md` for an example how to do that, instead of using "SLUG".